modules/thermal_hydraulics/src/utils/Numerics.C

```diff
@@ -78,6 +78,18 @@ Prandtl(ADReal cp, ADReal mu, ADReal k)
   return cp * mu / k;
 }
 
+Real
+Peclet(Real volume_fraction, Real cp, Real rho, Real vel, Real D_h, Real k)
+{
+  return volume_fraction * cp * rho * std::fabs(vel) * D_h / k;
+}
+
+ADReal
+Peclet(ADReal volume_fraction, ADReal cp, ADReal rho, ADReal vel, ADReal D_h, ADReal k)
+{
+  return volume_fraction * cp * rho * std::fabs(vel) * D_h / k;
+}
```

Comment:
This function should be templated on the type rather than have two identical formulations:
```
template <typename T>
T
Peclet(const & T volume_fraction, ...)
```

Reply:
Note this implementation will be moved to the header file since it's a template.

Reply:
@joshuahansel, for this one I was talking with Lise and she pointed that since all other dimensionless numbers are formatted in the same way I did for the Peclet number,  maybe it would be interesting to create a new PR and change all dimensionless numbers, writing them as templates

Reply:
Yeah, it's a probably a good idea, not a big deal either way.